### PR TITLE
Added direct proxy to allow redirecting a domain to another application.

### PR DESCRIPTION
**Pull request by Doc-Saintly**
In the case of hosting an application like myapp.mydomain.com and trying to redirect www.myapp.com -> myapp.mydomain.com it would always fail with a 502 gateway error.

This should fix that.

### CODE REVIEW

File: manifest.json [hunk 1]
```diff
@@ -61,6 +61,7 @@
                   "public_302" : "Visible redirect (302, temporary). Everybody will be able to access it.",
                   "public_301" : "Visible redirect (301, permanent). Everybody will be able to access it.",
                   "public_proxy": "Proxy, invisible (nginx proxy_pass). Everybody will be able to access it.",
+				  "public_direct": "Proxy, invisible direct (nginx proxy_pass). Everybody will be able to access it. No forward headers.",
```
Comment (kay0u) [inline manifest.json]: Just a small indentation problem here

```suggestion
                  "public_direct": "Proxy, invisible direct (nginx proxy_pass). Everybody will be able to access it. No forward headers.",
```

**Review by kay0u — APPROVED** [review]
Can you increment the package version please?

LGTM otherwise

**Comment by alexAubin** [comment]
This is a super late review, but I don't understand what issue this is addressing ... I don't understand why a regular redirect would end up in a 502. The new mode looks like a regular reverse proxy / proxy pass, but with less options enable and proxy_redirect not set to off ... Is that the root cause of the issue ? Would be nice to have a way to reproduce the initial issue ...

**Comment by zamentur** [comment]
@Doc-Saintly ?

**Comment by zamentur** [comment]
Related to https://github.com/YunoHost-Apps/redirect_ynh/issues/24

**Comment by alexAubin** [comment]
Closing due to lack of feedback and app got refactored

**Pull request closed without merging**